src/models/parallelCoordinates.js

```diff
@@ -38,11 +38,24 @@ nv.models.parallelCoordinates = function() {
             // Setup Scales
             x.rangePoints([0, availableWidth], 1).domain(dimensionNames);
 
+            var onlyNanValues = {};
             // Extract the list of dimensions and create a scale for each.
             dimensionNames.forEach(function(d) {
+                var extent = d3.extent(data, function(p) { return +p[d]; });
```

Comment:
oh so d is the value and not the index there?   different than jquery I guess :p   Okay, nevermind, calling it d is fine :)